_projects/_template.md

```diff
@@ -0,0 +1,50 @@
+---
+name: Example Astropy
+desc: This is the first idea for astropy
+# add a description of your project
+requirements:
+ - The proposal has to contain a clear concept.
+ - patch on any astropy affiliated package.
+difficulty: low
+issues:
+ - https://github.com/astropy/astropy/issue/4444
+ - https://github.com/astropy/astropy/issue/5555
+mentors:
+ - astrofrog
+initiatives:
+ - GSOC
+ - SOCIS
+tags:
+ - python
+collaborating_projects:
+ - astropy
+---
+This is an awesome project idea for Astropy.
+
+#### Description
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. Phasellus aliquam
+turpis neque, vel fermentum diam gravida id. Maecenas tempor magna libero, non
+faucibus dolor tincidunt non. Sed condimentum imperdiet odio sed volutpat.
+Mauris imperdiet viverra turpis a porttitor. Duis laoreet leo sit amet odio
+suscipit aliquet. Etiam vel malesuada est, in molestie nibh. Sed efficitur
+pharetra lorem, sed tempor sapien commodo ac. Vivamus sollicitudin pretium erat.
+Pellentesque luctus vehicula tortor, nec volutpat turpis euismod vel. Sed neque
+ipsum, imperdiet nec massa sit amet, accumsan lobortis augue. Donec luctus massa
+vitae velit scelerisque volutpat ut nec ex. Sed in velit tincidunt, volutpat
+turpis sit amet, dignissim nisi. Donec a mattis eros. Pellentesque vitae leo
+quam.
+
+#### Milestones
```

Comment:
Add a note along the lines of "note that Milestones are *not* necessary - just leaving a description is sufficient if you don't have milestones."  When I looked at this I assumed the milestones were mandatory, and only by looking at the README did I realize it's not...